slsSupportLib/src/Version.cpp

```diff
@@ -21,7 +21,8 @@ Version::Version(const std::string &s) {
 }
 
 bool Version::hasSemanticVersioning() const {
-    return version_ != defaultBranch_;
+
+    return (version_ != defaultBranch_[0]) && (version_ != defaultBranch_[1]);
```

Comment:
would you change this to defaultVersion? because defaultBranch does not make much sense anymore..